elasticsearch/src/test/java/org/hibernate/search/backend/elasticsearch/test/HostCanBeConfiguredIT.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.test;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.junit.Assert.fail;
+
+import java.net.ConnectException;
+
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.search.exception.SearchException;
+import org.hibernate.search.testsupport.TestForIssue;
+import org.junit.Test;
+
+/**
+ * Test to make sure that the ES host can be configured and not the default is applied unconditionally. To test it, the
```

Comment:
Looks like the "not" is misplaced.

Maybe "and the default is not applied unconditionally" would be better?